src/routes/blog/post/case-study-majik-kids/+page.markdoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Majik Kids: Building an audio-based content platform for children"
+description: Learn how Appwrite Cloud simplified their development process and accelerated productivity for Majik Kids as they worked 100s of artists, including musicians, voice artists, illustrators, etc., to build an alternative content platform with a focus on audio storytelling, creating a healthier learning experience for children.
```

Comment:
```suggestion
description: Learn how Appwrite Cloud simplified their development process and accelerated productivity for Majik Kids as they worked 100s of artists, including musicians, voice artists, and illustrators, to build an alternative content platform focusing on audio storytelling, creating a healthier learning experience for children.
```

Personal preference but I really dislike etc. in the middle of a sentence. It's also not necessary here to convey meaning